CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ A brief description of the categories of changes:
 
 ### Added
 
+* (py_wheel) Added `requires_file` and `extra_requires_files` attributes.
```

Comment:
nit: Please rebase and move the changelog note to the unreleased section, sorry for the extra noise here.

Reply:
My hope was that https://github.com/bazelbuild/rules_python/pull/1714#issuecomment-1904689271 would have been acceptable but alas, this file changed out from under me 😢